### PR TITLE
User details page

### PR DESCRIPTION
**Pull request by sonwit**
User detiails page 

## Description
Create user details page 

## Related Issue(s)
- #1008 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/features/amUI/userRightsPage/UserRightsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { amUIPath } from '@/routes/paths';
+
+export const UserRightsPage = () => {
+  const { t } = useTranslation();
+  const { id } = useParams();
+
+  const navigate = useNavigate();
+
+  const { data: reportee } = useGetReporteeQuery();
+  const { data: user } = useGetUserByUUIDQuery(id ?? '');
```
Comment (allinox) [inline src/features/amUI/userRightsPage/UserRightsPage.tsx:24]: Should we maybe use getPartyByUUIDQuery instead? Only people have userUuid and user profiles (which getUser fetches) but both people and orgs have partyUuid (userUuid = partyUuid for people) and that is the one we have stored in the browser now anyway. Besides, it saves us one abstraction-level for getting the name and other data 😊

Comment (allinox) [inline src/features/amUI/userRightsPage/UserRightsPage.tsx:24]: Also, do you think you can please add a couple of the people from Diskret Nær Tiger AS to the parties.json in the Mock project in the BFF? Just to make testing easier here and in the future? 🙏

You can for example add Diane Davis, Narnia Economics, and Santa Claus (just make sure that the partyuuid and other data is the same). That way we have the three main cases covered: Someone with roles in the org that is not you (Diane Davis), an org with rights (Narnia Economics), and someone that inherits rights through an org (Santa Claus).

File: src/features/amUI/users/UserItem/UserItem.module.css [hunk 2]
```diff
@@ -62,3 +62,8 @@
 .expandedContent {
   background-color: var(--fds-semantic-surface-neutral-subtle);
 }
+
+.clickableListitem {
```
Comment (allinox) [inline src/features/amUI/users/UserItem/UserItem.module.css:66]:
```suggestion
.clickable {
```
I suggest also renaming the expandable class in line 22 and 26 to `clickable` and then use that class on both the expandable items and the links. That way we get the nice hover and active colors too 😊

File: src/localizations/en.json [hunk 3]
```diff
@@ -200,6 +200,9 @@
     "user_search_placeholder": "Search for users",
     "user_no_search_result": "No users found"
   },
+  "user_rights_page": {
+    "page_title": "UserAccess - Altinn"
```
Comment (allinox) [inline src/localizations/en.json:204]:
```suggestion
    "page_title": "User access - Altinn"
```

**Review by allinox — APPROVED** [review]
Aaaand we're live with the next page! Great work!! 🎉

I left a few suggestions and requests here, but none of it is really crucial or complicated, so I'll just approve this anyway so you can add them and merge this at your convenience 😊

File: src/features/amUI/userRightsPage/UserRightsPage.tsx (hunk 1 above)
Comment (sonwit) [reply src/features/amUI/userRightsPage/UserRightsPage.tsx:24]: Absolutly!

File: src/features/amUI/common/PageContainer/PageContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        <div className={classes.pageActions}>
+          {onNavigateBack && (
+            <Button
+              variant='tertiary'
+              color='second'
+              onClick={onNavigateBack}
+              icon
+            >
+              <ArrowLeftIcon />
+              Tilbake
```
[the remark this answers is not in the document]
Comment (sonwit) [reply src/features/amUI/common/PageContainer/PageContainer.tsx:31]: 🫡

File: src/features/amUI/common/PageContainer/PageContainer.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+      <div className={classes.actionbar}>
+        <div className={classes.pageActions}>
+          {onNavigateBack && (
+            <Button
+              variant='tertiary'
+              color='second'
+              onClick={onNavigateBack}
+              icon
+            >
+              <ArrowLeftIcon />
```
[the remark this answers is not in the document]
Comment (sonwit) [reply src/features/amUI/common/PageContainer/PageContainer.tsx:30]: 👍

File: src/features/amUI/users/UserItem/UserItem.module.css (hunk 2 above)
Comment (sonwit) [reply src/features/amUI/users/UserItem/UserItem.module.css:66]: 👍

File: src/features/amUI/userRightsPage/UserRightsPage.tsx [hunk 6]
```diff
@@ -40,7 +40,7 @@ export const UserRightsPage = () => {
               size='sm'
               className={classes.heading}
             >
-              {user?.party.name}
+              {user?.name}
```
[the remark this answers is not in the document]
Comment (sonwit) [reply src/features/amUI/userRightsPage/UserRightsPage.tsx]: Yes. Forgot to check in file 🤦

**Pull request merged**